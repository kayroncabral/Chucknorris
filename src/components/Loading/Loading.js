import React from 'react'
import styles from './Loading.scss'

const Loading = () => {
  return (
    <header className={styles.Loading}>
      <p>Loading...</p>
    </header>
  )
}

export default Loading
