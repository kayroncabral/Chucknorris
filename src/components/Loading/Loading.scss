@import '../../styles/styles.scss';

.Loading {
  
}
