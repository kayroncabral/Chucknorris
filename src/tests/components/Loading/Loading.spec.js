import React from 'react'
import ReactShallowRenderer from 'react-test-renderer/shallow'
import Loading from '../../../components/Loading'

test('should render Header correctly', () => {
  const renderer = new ReactShallowRenderer()

  renderer.render(<Loading/>)
  expect(renderer.getRenderOutput()).toMatchSnapshot()
})
