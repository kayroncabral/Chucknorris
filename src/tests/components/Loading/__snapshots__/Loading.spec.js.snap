// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div
  className={undefined}
>
  <Loading
    color="#fff"
    delay={1000}
    height={64}
    type="cubes"
    width={64}
  />
</div>
`;
